features/fixtures/mazerunner/build.gradle

```diff
@@ -111,3 +112,6 @@ bugsnag {
     reportBuilds = false
     failOnUploadError = false
 }
+
+apply from: "../../../gradle/detekt.gradle"
```

Comment:
I don't think detekt/ktlint/lint run when building the mazerunner fixture. It'd be good to raise a separate task to make that so.